core/src/components/datetime/datetime.tsx

```diff
@@ -207,6 +208,11 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() isDateEnabled?: (dateIsoString: string) => boolean;
 
+  /**
+   * If `true`, the datetime will show the last days of the previous month and the first days of the next month on a table of 42 elements.
+   */
+  @Prop() showDaysOutsideCurrentMonth?: boolean = false;
```

Comment:
Can we change this to:

```suggestion
  @Prop() showAdjacentDays = false;
```

I think the shorter prop is a bit nicer.